tests/gdx-tests/src/com/badlogic/gdx/tests/UtfFontTest.java

```diff
@@ -23,9 +23,13 @@ public void create () {
 	public void render () {
 		Gdx.gl.glClear(GL20.GL_COLOR_BUFFER_BIT);
 		batch.begin();
-		font.draw(batch, "ÀÁÂÃÄÅÆÇÈÉÊËÌÍÎÏÐÑÒÓÔÕÖØŒŠþÙÚÛÜÝŸ\n" +
-				"àáâãäåæçèéêëìíîïðñòóôõöøœšÞùúûüýÿ\n", 20, 200);
-		font.draw(batch, "⌘  ¢ß¥£™©®ª×÷±²³¼½¾µ¿¶·¸º°¯§…¤¦≠¬ˆ¨‰", 20, 100);
+
+		 //https://github.com/libgdx/libgdx/pull/6501#issuecomment-821749417
+		font.draw(batch, "\u0089 \u0065 \u0089 \u0074", 20, 400);//Missing chars should print ' e t'
```

Comment:
This almost works, but the spaces between chars mean that kerning still isn't tested. Please remove the spaces, then your ordeal is over 😄. I'll merge then.

(It should be `"\u0089\u0065\u0089\u0074"`)

Reply:
done ! Tthanks for babysit  XD